configs/config.hardkernel_odroid_h4_btg

```diff
@@ -0,0 +1,68 @@
+CONFIG_LOCALVERSION="v0.9.0"
+CONFIG_OPTION_BACKEND_NONE=y
+CONFIG_VENDOR_HARDKERNEL=y
+CONFIG_VBOOT=y
+CONFIG_EDK2_CPU_THROTTLING_THRESHOLD_DEFAULT=10
+CONFIG_IFD_BIN_PATH="3rdparty/dasharo-blobs/$(MAINBOARDDIR)/descriptor.bin"
+CONFIG_ME_BIN_PATH="3rdparty/dasharo-blobs/$(MAINBOARDDIR)/me.bin"
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x100000
+CONFIG_PCIEXP_DEFAULT_MAX_RESIZABLE_BAR_BITS=37
+CONFIG_HAVE_IFD_BIN=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/bootsplash.bmp"
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_POWER_STATE_OFF_AFTER_FAILURE=y
+CONFIG_HAVE_ME_BIN=y
+CONFIG_DASHARO_PREFER_S3_SLEEP=y
+CONFIG_PCIEXP_SUPPORT_RESIZABLE_BARS=y
+CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
+CONFIG_DRIVERS_GENERIC_CBFS_SERIAL=y
+CONFIG_DRIVERS_GENERIC_CBFS_UUID=y
+# CONFIG_DRIVERS_MTK_WIFI is not set
+CONFIG_CBFS_VERIFICATION=y
+CONFIG_VBOOT_CBFS_INTEGRATION=y
+CONFIG_TPM2=y
+CONFIG_INTEL_TXT_BIOSACM_FILE="ADL_BIOSAC_v1_18_16_20230427_REL_NT_O1.PW_signed_256K.bin"
+CONFIG_INTEL_CBNT_SUPPORT=y
+CONFIG_INTEL_CBNT_LOGGING=y
+# CONFIG_INTEL_CBNT_GENERATE_KM is not set
+# CONFIG_INTEL_CBNT_GENERATE_BPM is not set
```

Comment:
Besides not generating the KM and BPM, we need to reserve space for the in CBFS so that the tools will replace them with proper manifests later during provisioning and signing. For that to happen we must include dummy manifests (empty 4K files). Example:

https://github.com/Dasharo/coreboot/commit/fc3babccccede26838bf826ffd19a340559838c0#diff-d63ca25a460a4aa425dcdc66c092c4df677aa2678c1de4e7e192ddfa2f3760ceR47-R48